feconf.py

```diff
@@ -348,6 +352,16 @@ def get_empty_ratings():
     },
 }
 
+# The list contains dicts with category of the classifier i.e.
+# 'StringClassifiers' or 'CodeClassifiers' and the IDs of the
+# corresponding classifiers.
+ANSWER_CLASSIFIER_CLASSES = [{
+    'category': 'StringClassifiers',
```

Comment:
The category here refers to StringClassifiers and CodeClassifiers. I think at some point this can help us to quickly know what are the StringClassifier and CodeClassifier algorithms available as opposed to putting everything in just one list.